### DISCUSSION TITLE
Typo in [@types/wordpress__block-editor] [@wordpress/block-editor]

### DISCUSSION THREAD

**Discussion by cpiber** [DefinitelyTyped/DefinitelyTyped#55345] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/da417026f068c0a194c28cf500742834da0b1da9/types/wordpress__block-editor/components/inner-blocks.d.ts#L48 should be `ButtonBlockAppender` (remove 'er')

[a comment by an automated account is not shown]

**Comment by dmsnell** [comment]
@cpiber are you interested in creating a Pull Request to propose this fix? that would be preferable to creating a new discussion for it

**Reply by cpiber** [reply]
Ok
